section8_function_inputs/assignments/1_paint_area_calculator.py

```diff
@@ -0,0 +1,46 @@
+"""
+You are painting a wall. The instructions on the paint can says that __1 can of paint can cover 5 square meters__ of wall. Given a random height and width of wall, calculate how many cans of paint you'll need to buy.
+
+number of cans = (wall height x wall width) ÷ coverage per can.
+
+e.g. Height = 2, Width = 4, Coverage = 5
+
+__number of cans = (2 * 4) / 5 = 1.6__
+
+But because you can't buy 0.6 of a can of paint, __the result should be rounded up to 2 cans__.
+
+IMPORTANT: Notice the name of the function and parameters must match those on line 13 for the code to work.
+
+### Example Input
+```
+test_h = 3
+test_w = 9
+```
+### Example Output
+```
+You'll need 6 cans of paint.
+```
+"""
+
+#Write your code below this line 👇
+import math
+
+def paint_calc(height, width, cover):
+   number_of_cans = (height * width)/cover                                                                #Find the number of cans           
+   actual_cans = math.ceil(number_of_cans)                                                         #Rounded up the number of can to find how many cans actually use    
+   print(f"You'll need {actual_cans} cans to paint this wall.")              
```

Comment:
Good choice of function, I prefer to combine lines 28 and 29 to the variable name like `cans_needed = math.ceil((height * width)/cover) `. But overall it is a good work already!

Reply:
As for improvement please handle the case when the function gets unexpected error like negative number or zero as well, just optional improvements